packages/react/src/components/participant/MediaTrack.tsx

```diff
@@ -4,6 +4,9 @@ import { useMediaTrackBySourceOrName } from '../../hooks';
 import { ParticipantClickEvent } from '@livekit/components-core';
 import { useEnsureParticipant } from '../../context';
 
+/**
+ * @deprecated Use `AudioTrack` or `VideoTrack` instead
```

Comment:
Perhaps a bit more drastic:
This component will be removed in the next version. Please use `AudioTrack` or `VideoTrack` instead.